js/data.js

```diff
@@ -89,6 +89,13 @@ const _CLEARING_TYPES = {
   }
 }
 
+const _PLAYERS = {
```

Comment:
I like what you are thinking here. Will make it easy to add the other mechanical variants :)